### PR TITLE
Autofixes issues using markdownlint-cli

### LINKED ISSUE #178: Run default fixes for markdownlint
Fix a large swath of markdown lint issues using an autofix feature

### PR DESCRIPTION
**Pull request by rozele**
Ran the following commands to generate these fixes:
```
npm i -g markdownlint-cli
markdownlint --fix *.md
markdownlint --fix **/*.md
markdownlint --fix **/**/*.md
```

The glob pattern in the markdownlint CLI didn't seem to work as expected, which is why I ran both `**` and **/**`.

Fixes #178

### CODE REVIEW

File: resources/templates/README.md [hunk 1]
```diff
@@ -11,7 +13,7 @@ Description of the project
 
 # Run it locally
 
-# ...
+#
```
Comment (TessFerrandez) [inline resources/templates/README.md]: Should we remove this completely or add another title?

File: team-agreements/minimalism-rules.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Follow Minimalism Rules 
+# Follow Minimalism Rules
 
-## (http://minifesto.org/)
+## (<http://minifesto.org/)>
```
Comment (TessFerrandez) [inline team-agreements/minimalism-rules.md]: While we’re fixing things, can we remove the ## here - this doesn’t make sense as a title, we can just leave it as a link

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 This project welcomes contributions and suggestions.  Most contributions require you to agree to a
 Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+the rights to use your contribution. For details, visit <https://cla.microsoft.com.>
```
Comment (rozele) [inline CONTRIBUTING.md]: fix link closures.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -66,9 +66,9 @@ see the [LICENSE](LICENSE) file, and grant you a license to any code in the repo
 Microsoft, Windows, Microsoft Azure and/or other Microsoft products and services referenced in the documentation
 may be either trademarks or registered trademarks of Microsoft in the United States and/or other countries.
 The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
-Microsoft's general trademark guidelines can be found at http://go.microsoft.com/fwlink/?LinkID=254653.
+Microsoft's general trademark guidelines can be found at <http://go.microsoft.com/fwlink/?LinkID=254653.>
```
Comment (rozele) [inline CONTRIBUTING.md]: Move `>` inside period.

File: team-agreements/minimalism-rules.md (hunk 2 above)
Comment (rozele) [reply team-agreements/minimalism-rules.md]: I'm only running autofixing in this PR.

File: resources/templates/README.md (hunk 1 above)
Comment (rozele) [reply resources/templates/README.md]: I've updated the PR to include the rule we discussed (`no-trailing-punctuation: false`)

**Pull request merged**